index.js

```diff
@@ -151,6 +151,19 @@ MetaInspector.prototype.getMetaDescription = function()
 	return this;
 }
 
+MetaInspector.prototype.elemContainsTag = function(elem,tag) {
+	if (elem.name && elem.name === tag ) {
```

Comment:
while doing tests I did not see the property tagName only  name.  Name change would be good.